src/fondant/cli.py

```diff
@@ -28,15 +28,9 @@
 from pathlib import Path
 from types import ModuleType
 
-from fondant.build import build_component
-from fondant.compiler import DockerCompiler, KubeFlowCompiler, VertexCompiler
-from fondant.component import BaseComponent, Component
-from fondant.executor import ExecutorFactory
-from fondant.explorer import (
-    run_explorer_app,
-)
-from fondant.pipeline import Pipeline
-from fondant.runner import DockerRunner, KubeflowRunner, VertexRunner
+if t.TYPE_CHECKING:
```

Comment:
what does this do?

Reply:
It only imports the guarded imports during type checking, not at runtime. See [docs](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING).